src/app/pages/SearchResultsPage/hooks.ts

```diff
@@ -182,6 +215,10 @@ export const useSearch = (q: SearchParams) => {
     .map(l => l.evm_tokens)
     .flat()
     .sort((t1, t2) => t2.num_holders - t1.num_holders)
+  const proposals = queries.proposals.results
+    .map(l => l.proposals)
+    .flat()
+    .filter(getProposalTextMatcherFor(q.networkProposalNameFragment))
```

Comment:
Filtering should happen inside useNetworkProposalsConditionally, or even create useGetConsensusProposalsByName

Reply:
> Filtering should happen inside useNetworkProposalsConditionally, or even create useGetConsensusProposalsByName

OK, created `useGetConsensusProposalsByName()`.